source/sdk/lang/Backtrace.ooc

```diff
@@ -52,13 +53,19 @@ BacktraceHandler: class {
         } else {
             // fall back on execinfo? still informative
             version (linux || apple) {
-                stderr write("[lang/Backtrace] Falling back on execinfo.. (build extension if you want fancy backtraces)\n")
+                if (!WARNED_ABOUT_FALLBACK) {
```

Comment:
I'm pretty sure the name is arbitrary. However, I felt that we should strive to be as close to the original SDK as possible in files we're not messing around in (read: improving) yet.